uint8   robot_id
uint8   topic_id
builtin_interfaces/Time ts_limit
---
uint8[] headers
